src/modules/auth/index.route.ts

```diff
@@ -0,0 +1,9 @@
+import { FastifyPluginAsyncTypebox } from '@fastify/type-provider-typebox'
+import createAuthController from './controllers/index.js'
+
+const authRoutes: FastifyPluginAsyncTypebox = async (fastify) => {
+  const authController = createAuthController(fastify)
+  fastify.post('', {}, authController.login)
```

Comment:
This is an antipattern. You are splitting the route code from its definitions, and you shouldn't really do that at all, as they are tightly coupled together.